recipes/multi-hauser/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:latest
```

Comment:
I'm pretty science dog on the docker stuff -- is there someone else that could look at this too? I mean I trust it works and its just a recipe, so not a huge deal either way.

Reply:
@claygoddard FWIW, I had written a similar dockerfile.  I like Cam's version more because it delegates construction of the artifacts to the container itself.  Previously, I had a build script that then pushed the output to the container.  FullStory customers have also created similar approaches https://github.com/fullstorydev/hauser/pull/12/files and https://github.com/aclaimant/docker-hauser/blob/master/Dockerfile.  @camphillips22, with this recipe, do you think that Schuylr's PR should be closed in reference to this?

Reply:
I guess it could be. Honestly, I think that providing the binaries obviates the need for a dedicated docker image since it's easy enough to add the binary to the image or just to curl it like this. Especially since hauser depends on setting up Auth and using secrets and that can be quite different depending on the deployment environment. Maybe I could add a one-liner to the main README with a command to add the latest version to a Dockerfile.